data/remnant missions.txt

```diff
@@ -1716,3 +1725,164 @@ mission "Remnant: Broken Jump Drive job"
 		dialog "The research team quickly offloads the broken jump drive and pays you <payment> before hurrying back to their lab."
 
 
+
+mission "Remnant: Face to Maw 1"
+	name "Talk to Plume about the void sprites"
+	description "Plume would be very interested in a report on what happened with the void sprites."
+	landing
+	to offer
+		has "remnant: return the samples timer"
+	source
+		system "Nenia"
+	destination "Aventine"
+	on offer
+		conversation
+			`As the swirls of gas close over the <ship> you wonder idly what brought you back here. This time, there is no apologetic researcher asking you to risk your life for research, or to return eggs from where they were stolen. You catch a glimpse of the Archon materializing above the clouds as it decloaks, letting you know that you're being watched, but saying nothing; for now, it is nothing more than a silent, brooding presence in the skies of Nenia. The silent depths of an alien world fold over you, gradually erasing the rest of the galaxy from sight.`
+			`	As the <ship> levels off at the designated altitude, you strap yourself into a pressure suit. Stepping into the airlock, you grimace at the antiquity of these "hard-shell" shield systems, and lower the shields so that you can exit onto the hull. Out on the surface, you admire the misty kaleidescope of colors filtering down through the cloud layers, playing over distant shapes moving in the mists.`
```

Comment:
```suggestion
			`	As the <ship> levels off at the designated altitude, you strap yourself into a pressure suit. Stepping into the airlock, you grimace at the antiquity of these "hard-shell" shield systems, and lower the shields so that you can exit onto the hull. Out on the surface, you admire the misty kaleidoscope of colors filtering down through the cloud layers, playing over distant shapes moving in the mists.`
```